samples/winapi_drivelist/main.c

```diff
@@ -9,45 +9,47 @@ int main(void)
 
     // Mount some drives for demonstration purposes
     BOOL ret;
+    DWORD error;
     ret = nxMountDrive('C', "\\Device\\Harddisk0\\Partition2\\");
     if (!ret) {
-        debugPrint("Failed to mount C: drive!\n");
+        // Additional error info can be retrieved with GetLastError()
+        error = GetLastError();
+        debugPrint("Failed to mount C: drive! Error code: %x\n", error);
```

Comment:
Here you use:
```
         debugPrint("Failed to mount C: drive! Error code: %x\n", error); 
```

In other instances you use:

```
         debugPrint("Couldn't unmount C: drive! Reason: %x", error); 
```

Reply:
This was not done. In one file you use "Reason" in the other one you use "Error code".
I didn't check if "Couldn't" vs. "Failed to" is also still an issue.

Reply:
This is still the case.